Description: Include the Cython headers from the source directory
 So they are found when running the tests without installing Sage.
Author: Tobias Hansen <thansen@debian.org>

--- a/sage/src/sage/misc/cython.py
+++ b/sage/src/sage/misc/cython.py
@@ -232,7 +232,7 @@
     additional_source_files, s = parse_keywords('cfile', s)
 
     v, s = parse_keywords('cinclude', s)
-    inc = [environ_parse(x.replace('"','').replace("'","")) for x in v] + sage_include_directories()
+    inc = [environ_parse(x.replace('"','').replace("'","")) for x in v] + sage_include_directories(use_sources=True)
     s = """\ninclude "cdefs.pxi"\n""" + s
     s = """\ninclude "cysignals/signals.pxi"  # ctrl-c interrupt block support\ninclude "stdsage.pxi"\n""" + s
     args, s = parse_keywords('cargs', s)
